src/rpdk/core/data_loaders.py

```diff
@@ -176,36 +182,45 @@ def load_resource_spec(resource_spec_file):  # pylint: disable=R # noqa: C901
                                 "maximum",
                                 "exclusiveMinimum",
                                 "exclusiveMaximum",
+                                "multipleOf",
```

Comment:
cant leave the comment above but could u take this structure out and create a static `_dict_`? it is really hard to read this way.

Reply:
I support this idea too. Also can you create `type_specific_keywords` from the `_dict_` you create for this structure so that all keywords specified in the `_dict_` will be automatically available in `type_specific_keywords `

Reply:
Started with dictionary, but [`integer` and `number` types](https://json-schema.org/understanding-json-schema/reference/numeric.html) share JSON schema keywords, and lists cannot be used as Python dictionary keys, so I switched

Switched `type_specific_keywords` to be automatically generated from the mapping :)